pkgs/development/python-modules/glean-sdk/default.nix

```
@@ -57,6 +65,10 @@ buildPythonPackage rec {
     "test_flipping_upload_enabled_respects_order_of_events"
   ];
 
+  postInstallCheck = lib.optionalString (stdenv.hostPlatform.parsed.kernel.execFormat == lib.systems.parse.execFormats.elf) ''
+    readelf -a $out/${python.sitePackages}/glean/libglean_ffi.so | grep -F 'Shared library: [liblmdb.so'
```

```suggestion
    readelf -a $out/${python.sitePackages}/glean/libglean_ffi.so | grep -qF 'Shared library: [liblmdb.so'
```
don't we need ``-q`` here?